Library/Koala/KoalaTests.swift

```diff
@@ -590,4 +590,52 @@ final class KoalaTests: TestCase {
     XCTAssertEqual(["live_stream_live"], client.properties(forKey: "live_stream_state", as: String.self))
     XCTAssertEqual(["Cool Live Stream"], client.properties(forKey: "live_stream_name", as: String.self))
   }
+
+  func testTrackViewedPaymentMethods() {
+    let client = MockTrackingClient()
+    let koala = Koala(client: client)
+
+    koala.trackViewedPaymentMethods()
+    XCTAssertEqual(["Viewed Payment Methods"], client.events)
+  }
+
+  func testTrackViewedAddNewCard() {
+    let client = MockTrackingClient()
+    let koala = Koala(client: client)
+
+    koala.trackViewedAddNewCard()
+    XCTAssertEqual(["Viewed Add New Card"], client.events)
+  }
+
+  func testTrackDeletedPaymentMethod() {
+    let client = MockTrackingClient()
+    let koala = Koala(client: client)
+
+    koala.trackDeletedPaymentMethod()
+    XCTAssertEqual(["Deleted Payment Method"], client.events)
+  }
+
+  func testTrackErroredDeletePaymentMethod() {
```

Comment:
We should also rename the test here. Feel free to simply accept the suggestion through Github

```suggestion
  func testTrackDeletePaymentMethodError() {
```

Reply:
Ops. missed that one